packages/terra-hookshot/tests/wdio/hookshot-combinations-spec.js

```diff
@@ -0,0 +1,892 @@
+/* global browser, Terra, before */
+
+describe('[1] Display correctly for contentAttachment="top start" & targetAttachment="top start"', () => {
```

Comment:
Given the nature of how the screenshots are named, it might be better to simplify these names?